Cython/Compiler/Nodes.py

```diff
@@ -3128,6 +3126,7 @@ def as_cfunction(self, cfunc=None, scope=None, overridable=True, returns=None, e
             if scope is None:
                 scope = cfunc.scope
             cfunc_type = cfunc.type
+            has_explicit_exc_clause=True
```

Comment:
Are the warnings caused by this change?

I don't remember why I added this here. I don't have it in our void linux package (https://github.com/void-linux/void-packages/blob/master/srcpkgs/python3-Cython/patches/fix-regression.patch).

Reply:
Yeah it does look like it. I'll see if I can come up with a test for the warnings.

Reply:
https://github.com/cython/cython/pull/6430 should deal with it I think